ProjectFiles/Assets/Scripts/Gameplay/ParkingSpaceManager.cs

```diff
@@ -33,6 +33,16 @@ public ParkingSpaceManager()
                 Debug.Log($"Space Added, SpaceID: {parkingSpace.SpaceID}.");
             }
         }
+
+        public List<Transform> GetSpaceTransforms()
+        {
+            List<Transform> transforms = new List<Transform>();
+            foreach (var space in parkingSpaces)
+            {
+                transforms.Add(space.Value.transform);
```

Comment:
I think this can be a one-line
```C#
return parkingSpaces.Values.Select(s => s.transform).toList();
``` 
;)

Reply:
ok nerd

Reply:
we should probably just store the transform of the space in itself on creation cos we said doing .transform might be kinda expensive and I also do it in the function to generate the parking spaces for the round. spaces don't move so it should be fine